Utils/hdinsight-node-common/src/com/microsoft/azure/sparkserverless/serverexplore/sparkserverlessnode/SparkServerlessClusterRootModuleImpl.java

```diff
@@ -36,6 +43,8 @@
     // TODO: determine root node name
     private static final String BASE_MODULE_NAME = "Azure Data Lake Spark Pool";
 
+    private static final String SPARK_NOTEBOOK_LINK = "https://adsnotebookpreview.southeastasia.cloudapp.azure.com/";
```

Comment:
Can we replace it with an aka.ms short redirection url, such as `aka.ms/spkadlnb` ?

Reply:
Done.